src/database/models/index.js

```diff
@@ -41,4 +41,4 @@ Object.keys(db).forEach(modelName => {
 db.sequelize = sequelize;
 db.Sequelize = Sequelize;
 
-export default db;
```

Comment:
Thank you, work is in progress to implement the feedback received on this review.
However, it seems this could have been an automatic effect from `git rebase`. I will look into it going forward